warehouse/views.py

```diff
@@ -189,12 +208,12 @@ def search(request):
     available_filters = collections.defaultdict(list)
 
     for cls in request.db.query(Classifier).order_by(Classifier.classifier):
-        first, *_ = cls.classifier.split(' :: ')
+        first = cls.classifier.split(' :: ')[0]
```

Comment:
Mostly because the other variables are never used. I'm happy to change it back tho.